apps/dashboard/src/pages/organization-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { OrganizationList as OrganizationListForm } from '@clerk/clerk-react';
+import { PageMeta } from '@/components/page-meta';
+import { ROUTES } from '@/utils/routes';
+
+export const OrganizationList = () => {
+  return (
+    <>
+      <PageMeta title="Select or create organization" />
+      <OrganizationListForm
+        appearance={{
+          elements: {
+            organizationAvatarUploaderContainer: {
+              display: 'none',
+            },
+          },
+        }}
+        hidePersonal
+        skipInvitationScreen
+        afterSelectOrganizationUrl={ROUTES.WORKFLOWS}
+        afterCreateOrganizationUrl={ROUTES.WORKFLOWS}
```

Comment:
Redirect to `/workflows` after org is created/picked, we might change that in the future when will be having the new onboarding flow.